xcode/Subconscious/Shared/Components/Notebook/NotebookNavigationView.swift

```diff
@@ -86,6 +86,15 @@ struct NotebookNavigationView: View {
                         Image(systemName: "gearshape")
                     }
                 }
+                ToolbarItem(placement: .navigationBarLeading) {
```

Comment:
Unsure what I prefer here. Might be better to have both icons in the top right corner?